taxi/models.py

```diff
@@ -15,7 +15,7 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(max_length=255, unique=True, )
```

Comment:
This comma is redundant 
```suggestion
    license_number = models.CharField(max_length=255, unique=True)
```